mteb/tasks/PairClassification/por/Assin2RTE.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class Assin2RTE(AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="assin2-rte",
+        dataset={
+            "path": "assin2",
+            "revision": "0ff9c86779e06855536d8775ce5550550e1e5a2d",
+        },
+        description="Recognizing Textual Entailment part of the ASSIN 2, an evaluation shared task collocated with STIL 2019.",
+        reference="https://link.springer.com/chapter/10.1007/978-3-030-41505-1_39",
+        type="PairClassification",
+        category="s2s",
+        eval_splits=["test"],
```

Comment:
we can add the validation sest also ?
```suggestion
        eval_splits=["validation", "test"],
```

Reply:
as the test set is already big enough (>2048), we can keep test only if that's fine with you

Reply:
okay great!